src/main/kotlin/com/whatever/domain/auth/service/AuthService.kt

```diff
@@ -75,4 +98,11 @@ class AuthService(
             refreshToken,
         )
     }
+}
+
```

Comment:
```suggestion
const val OIDC_PUBLIC_KEY = oidc-public-key
```
요런건 상수화 해둬도 좋을 것 같습니다!

Reply:
오오 이런 추천은 처음봤네요.
안그래도 상수화가 고민되긴 했는데 어느 위치가 적절할지 여쭤보고 싶었습니다!
다양한 파일들에서 사용되어서 고민이 되네요,,